cmd/snap/cmd_recovery.go

```diff
@@ -56,10 +56,21 @@ func init() {
 }
 
 func notesForSystem(sys *client.System) string {
+	var notes []string
+
 	if sys.Current {
-		return "current"
+		notes = append(notes, "current")
+	}
+
+	if sys.DefaultRecoverySystem {
+		notes = append(notes, "default-recovery-system")
```

Comment:
this could just be "default-recovery", as they are all systems and it makes it a bit shorter?